src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.baseStyles.tsx

```diff
@@ -34,13 +21,6 @@ export const baseStyles = (theme: SuomifiTheme) => css`
   }
 
   &.fi-multiselect--open {
-    & .fi-filter-input_input-element-container {
-      &:before {
-        border-color: transparent transparent ${theme.colors.blackBase}
-          transparent;
-        border-width: 0 4px 6px 4px;
-      }
-    }
```

Comment:
I think you mentioned about the removal of some styles, but not sure was it this PR.
Can you tell again why this style is no longer needed for me and maybe for other reviewers also.

Reply:
This is actually from a commit that was in the previous PR #600. That pseudo element was the CSS-based caret-icon